go.work.sum

```diff
@@ -262,6 +262,7 @@ github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802 h1:1BDTz0u9nC3//pOC
 github.com/DataDog/datadog-go v3.2.0+incompatible h1:qSG2N4FghB1He/r2mFrWKCaL7dXCilEuNEeAn20fdD4=
 github.com/DataDog/zstd v1.4.5/go.mod h1:1jcaCB/ufaK+sKp1NBhlGmpz41jOoPQ35bpF36t7BBo=
 github.com/GoogleCloudPlatform/cloudsql-proxy v1.22.0 h1:aTDBS16pX1X4ZR/GFsC2NcOCYJ1hDJwJm3WmKRA905Q=
+github.com/Groxx/cadence-idl v0.0.0-20240325232920-032063621ac4/go.mod h1:oyUK7GCNCRHCCyWyzifSzXpVrRYVBbAMHAzF5dXiKws=
```

Comment:
I assume you're going to update this?

Reply:
Thought I had, but apparently not - will fix

Reply:
still not fixed

Reply:
yep, just cleaned it up now.
`go mod tidy` and `go work sync` are for some reason not removing it :\  no idea why.